error Arguments to path.join must be strings

I am getting the following error when installing gulp-phantomcss:

```
> gulp-phantomcss@0.1.0 postinstall /Users/dev/node_modules/gulp-phantomcss
> ./node_modules/bower/bin/bower install phantomcss

bower phantomcss#~0.10      not-cached git://github.com/Huddle/PhantomCSS.git#~0.10
bower phantomcss#~0.10         resolve git://github.com/Huddle/PhantomCSS.git#~0.10
bower phantomcss#*          not-cached git://github.com/Huddle/PhantomCSS.git#*
bower phantomcss#*             resolve git://github.com/Huddle/PhantomCSS.git#*
bower phantomcss#~0.10           error Arguments to path.join must be strings

Stack trace:
TypeError: Arguments to path.join must be strings
    at Object.posix.join (path.js:488:13)
    at GitHubResolver._checkout (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/lib/core/resolvers/GitHubResolver.js:54:21)
    at /Users/dev/node_modules/gulp-phantomcss/node_modules/bower/lib/core/resolvers/GitResolver.js:69:21
    at _fulfilled (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:798:54)
    at self.promiseDispatch.done (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:827:30)
    at Promise.promise.promiseDispatch (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:760:13)
    at /Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:574:44
    at flush (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:108:17)
    at process._tickCallback (node.js:355:11)

Console trace:
Trace
    at StandardRenderer.error (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/lib/renderers/StandardRenderer.js:74:17)
    at Logger.<anonymous> (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/bin/bower:109:18)
    at Logger.emit (events.js:107:17)
    at Logger.emit (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/bower-logger/lib/Logger.js:29:39)
    at /Users/dev/node_modules/gulp-phantomcss/node_modules/bower/lib/commands/install.js:27:16
    at _rejected (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:808:24)
    at /Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:834:30
    at Promise.when (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:1079:31)
    at Promise.promise.promiseDispatch (/Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:752:41)
    at /Users/dev/node_modules/gulp-phantomcss/node_modules/bower/node_modules/q/q.js:574:44

System info:
Bower version: 1.2.8
Node version: 0.12.2
OS: Darwin 14.4.0 x64
npm ERR! Darwin 14.4.0
npm ERR! argv "node" "/Users//.npm-packages/bin/npm" "install"
npm ERR! node v0.12.2
npm ERR! npm  v2.13.0
npm ERR! code ELIFECYCLE

npm ERR! gulp-phantomcss@0.1.0 postinstall: `./node_modules/bower/bin/bower install phantomcss`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gulp-phantomcss@0.1.0 postinstall script './node_modules/bower/bin/bower install phantomcss'.
npm ERR! This is most likely a problem with the gulp-phantomcss package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./node_modules/bower/bin/bower install phantomcss
npm ERR! You can get their info via:
npm ERR!     npm owner ls gulp-phantomcss
npm ERR! There is likely additional logging output above.
```

Have you seen this before?

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "through2": "~2.0",
     "lodash": "~3.0",
     "phantomjs": "~1.9",
-    "bower": "~1.2"
+    "bower": "~1.3.8"
   },
   "devDependencies": {
     "mocha": "~2.2"

